extensions/amp-list/0.1/amp-list.js

```diff
@@ -894,6 +915,24 @@ export class AmpList extends AMP.BaseElement {
     if (!this.loadMoreFailedElement_) {
       this.loadMoreFailedElement_ = childElementByAttr(
           this.element, 'load-more-failed');
+
+      if (!this.loadMoreFailedElement_) {
+
+        this.loadMoreFailedElement_ = htmlFor(this.win.document)`
+          <div load-more-failed class="i-amphtml-default-ui">
+            <div class="i-amphtml-amp-list-load-more-message">
+              Unable to Load More
+            </div>
+            <button class=" i-amphtml-amp-list-load-more-button
```

Comment:
Nit: extra space.

Reply:
This was to make it line with nicely with the long classnames below it, but I can take it out. :)